felix/fv/bpf_test.go

```diff
@@ -945,6 +1046,30 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 
 					pol = createPolicy(pol)
 
+					By("allowing to self via MASQ", func() {
```

Comment:
I only wonder why we had to add this policy. I'd expect that stuff from the host is allowed. It is not needed when CTLB is used as it is always resolved at connect time and such connections never leave the pod.